index.html

```diff
@@ -0,0 +1,117 @@
+<div id="menu" class="main-nav">
+    
+        <ul id="menu-ul" class="d-flex main-nav-ul">
+        <li id="close_menu" class="close_menu">x</li>
+        <li class="top-nav-list"><a href="/about.html">About</a></li>
+        <li class="top-nav-list currunt-tab"><a href="/index.html">Program</a></li>
+        <li class="top-nav-list">Join</li>
+        <li class="top-nav-list">Sponsor</li>
+        <li class="top-nav-list">News</li>
+       </ul>
+    </div>
+<div class="home-contanier">
+           <h1 class="home-title">Space Tech Expo 2021</h1>
+
+           <h2 class="home-haeder">SPACE TECH EXPO EUROPE RETURNS FOR 2021</h2>
+
+           <p class="home-p">
+            Register now for Europe’s largest exhibition and conference for the space industry  
+        </p>
+        <p class="date">2021.10.27(THU)~ 30(FRI)</p>
+        <p class="place">@Findorffstraße 101 28215 Bremen, Germany</p>
+        </div>
+        </section>
+
+        <section class="main-page">
+<h2 class="main-program">Main Program</h2>
+<hr class="red-line">
+<div class="main-page-items">
+<div class="program-item"> <img class="program-img" src="./img/lechure.png" alt="Lecture"> <span class="program-name">Lecture</span> <p class="program-p">Listen to the speakers from variuos countries about the massegae of sharing and opening.</p> 
+ </div>
+ <div class="program-item"> <img class="program-img" src="./img/layer.png" alt="Exhibition"> <span class="program-name">CC Exhibition</span> <p class="program-p">Appreciate various creaion applying CC lincense of artists, organized from Art Center Nabi.</p> 
+ </div>
+ <div class="program-item"> <img class="program-img" src="./img/multiple.png" alt="Forum"> <span class="program-name">Forum</span> <p class="program-p">Have the time to share your thoughts and opinions with experts for each topic.</p> 
+ </div>
+ <div class="program-item"> <img class="program-img" src="./img/business.png" alt="Workshop"> <span class="program-name">Workshop</span> <p class="program-p">Try creating your own work using open source license rather than just watching.</p> 
+ </div>
+</div>
+
+ <button type="button" class="main-btn text-white">Join Space Tech Expo 2021</button>
+    </section>  
+
+    <section class="speakers">
+<h2 class="speaker-main">Featured Speakers</h2>
+<hr class="red-line">
+<div id="speakers" class="card-all"></div>
+    </section>
+
+    <section class="sponsors">
+        <h2 class="main-program">Partners</h2>
+<hr class="red-line">
+<div class="sponsers-list">
+    <div class="sponser1"></div>
+    <div class="sponser2"></div>
+    <div class="sponser3"></div>
+    <div class="sponser4"></div>
+    <div class="sponser5"></div>
+</div>
+    </section>
+    <footer class="end">
+        <img class="footer-logo" src="./img/footer-logo.png" alt="Logo">
+        <p class="footer-p">2021 Creative Commons Germany.<br> Some Rights Reserved.</p>
+    </footer>
+    <script src="./js/main.js"></script>
```

Comment:
- [ ] On the mobile screen the footer of the home page is expected to look like the given design with just the `More` text having a white background.

| Original | Yours |
| ------- | ----- |
| <img src="https://user-images.githubusercontent.com/33323077/133306206-e7583dc4-4504-47a0-96fa-0c5568e85c92.png" height="250px"/> | <img src="https://user-images.githubusercontent.com/33323077/133306298-0c22ff3a-bbf7-4a85-ab50-1da0039f49ab.png" height="250px"/>